Update 8/23/17

mjfre opened the pull request:
Add RobotTreasureHunt
Fixed typos

Commit:
Update 8/23/17

Add RobotTreasureHunt
Fixed typos

keithagroves reviewed (commented):
You should add robot square back!

Commit:
Update 8/26/2017

Added Quiz Game recipe
Added Four Square recipe
Deleted answers from RobotTreasureHunt
Added the background image for RTH
Fixed some formatting inconsistencies

Commit:
8/29/17

Added RobotSquare to day2 package

Commit:
8/30/17 Update

Fixed SmartShapes
Fixed RobotTreasureHunt (hopefully)

TODO: Add processing recipes or create new repository for them

Commit:
9/12/17 Update

Added Processing recipes for day 1

Commit:
9/12/17

Renamed packages to maintain correct order

Added Tash Me! Recipe

MORE RECIPES COMING SOON

Commit:
More day1 recipes

Added more day1 recipes
Added RobotsInSpace and accompanying files
Added “handouts” for StarShow and FlamingNinjaStar

Commit:
Recipes from PDF to HTML

Commit:
Changed file names
